ssg/build_yaml.py

```diff
@@ -1197,6 +1198,12 @@ def _make_items_product_specific(self, items_dict, product_suffix, allow_overwri
                 continue
 
             label = full_label.split("@")[0]
+
+            if label in Rule.GLOBAL_REFERENCES:
+                raise ValueError(
+                    "You cannot use product-qualified for '{item_u}' reference. Please remove the product-qualifier and merge values with existing reference. Original line: {item_q}: {value_q}"
```

Comment:
@ggbecker If you want to fix this up to pass PEP8, I'd be happy to merge it.

Reply:
Done. Let's wait for test results.